wagtail_localize/machine_translators/tests/test_libretranslate_translator.py

```diff
@@ -0,0 +1,109 @@
+from unittest import mock
+
+from django.test import TestCase, override_settings
+from wagtail.models import Locale
+
+from wagtail_localize.machine_translators import get_machine_translator
+from wagtail_localize.machine_translators.libretranslate import LibreTranslator
+from wagtail_localize.strings import StringValue
+
+
+LIBRETRANSLATE_SETTINGS_ENDPOINT = {
+    "CLASS": "wagtail_localize.machine_translators.libretranslate.LibreTranslator",
+    "OPTIONS": {
+        "LIBRETRANSLATE_URL": "https://libretranslate.org",
+        "API_KEY": "test-api-key",
+    },
+}
+
+
+class TestLibreTranslator(TestCase):
+    @override_settings(
+        WAGTAILLOCALIZE_MACHINE_TRANSLATOR=LIBRETRANSLATE_SETTINGS_ENDPOINT
+    )
+    def setUp(self):
+        self.english_locale = Locale.objects.get()
+        self.french_locale = Locale.objects.create(language_code="fr-fr")
```

Comment:
thank you for changing this. You are missing a test for `LibreTranslator.language_code()`:

```python
def test_language_code(self):
    self.assertEqual(self.translator.language_code(self.english_locale.language_code)), "en")
    self.assertEqual(self.translator.language_code(self.french_locale.language_code)), "fr")
    self.assertEqual(self.translator.language_code("foo-bar-baz"), "foo")
```

Reply:
added it